spec/system/targets/show_spec.rb

```diff
@@ -362,7 +361,10 @@
       scenario 'student can resubmit non-resubmittable target if its failed' do
         # Make the first failed submission the latest, and the only one.
         submission_2.destroy!
-        submission_1.update(latest: true)
+
+        # rubocop:disable Rails/SkipsModelValidations
+        submission_1.timeline_event_owners.update_all(latest: true)
+        # rubocop:enable Rails/SkipsModelValidations
```

Comment:
Use single-line comment to disable Rubocop.

```suggestion
        submission_1.timeline_event_owners.update_all(latest: true) # rubocop:disable Rails/SkipsModelValidations
```